node/rpc/src/state_api.rs

```diff
@@ -123,9 +123,9 @@ pub(crate) async fn state_market_deals<
     let ts = data.chain_store.tipset_from_keys(&tsk)?;
     let actor = data
         .state_manager
-        .get_actor(&market::ADDRESS, *ts.parent_state())?
+        .get_actor(&market::ADDRESS.into(), *ts.parent_state())?
```

Comment:
We can use the new constants, right?

```suggestion
        .get_actor(&Address::MARKET_ACTOR, *ts.parent_state())?
```

Reply:
fixed